services/deprecated-service.js

```diff
@@ -4,7 +4,7 @@ const BaseService = require('./base')
 const { Deprecated } = require('./errors')
 
 // Only `url` is required.
-function deprecatedService({ url, label, category, examples = [] }) {
+function deprecatedService({ url, label, category, examples = [], message }) {
```

Comment:
I feel like this change ought to have a test in `deprecated-service.spec.js`, though we don't yet seem to have that file yet. We're testing it all indirectly through the services, so it seems more like a nice to have than a blocker.

I'd like to get to a point where we can make a test suite that hits 100% coverage on the core code without running the service tests. It should let us refactor the core code with more confidence and speed, certain that nothing has broken. As things are now a lot of functionality has to be tested through the services which is fairly slow, and makes it especially harder for newer contributors to change things in core. I was surprised to realize the other day that this project is 40k lines of code, and it's just unrealistic to expect people to have read most of it, to know how all the services may be using various parts of the core. But that's a bigger project for another day.

Reply:
> I was surprised to realize the other day that this project is 40k lines of code

WOAH! did not know that

Reply:
Let me take a look to see how easily tests could be added for the deprecated service def.